src/Pact/Typechecker.hs

```diff
@@ -932,7 +938,7 @@ toAST (TObject Term.Object {..} _) = do
   ty <- TySchema TyObject <$> traverse toUserType _oObjectType <*> pure FullSchema
   Object <$> (trackNode ty =<< freshId _oInfo "object")
     <*> mapM toAST _oObject
-toAST TConst {..} = toAST (_cvRaw _tConstVal) -- TODO typecheck here
+toAST TConst {..} = toAST (_cvEval _tConstVal) -- TODO typecheck here
```

Comment:
re: the existing `TODO`, I'm not so sure exactly what needs to be done here.

Reply:
well, I guess it's possible to write `(defconst FOO:string 1)`